packages/xod-client-electron/src/app/workspaceActions.js

```diff
@@ -261,30 +272,22 @@ export const onIDELaunch = R.curry(
       ensureWorkspace,
       updateWorkspace(send, oldPath),
       () => {
-        const maybePatchToOpen = getFileToOpen();
-
-        if (Maybe.isNothing(maybePatchToOpen)) {
-          const isFirstLaunch = R.isEmpty(oldPath);
-          const projectName = isFirstLaunch
-            ? 'welcome-to-xod'
-            : 'welcome-again';
-          return onOpenBundledProject(
-            send,
-            updateProjectPath,
-            projectName
-          ).then(() => {
-            // TODO: better pass it along the project with REQUEST_SHOW_PROJECT event
-            if (!isFirstLaunch) {
-              send(EVENTS.PAN_TO_CENTER);
-            }
-          });
+        const maybeProjectPathToOpen = getFileToOpen();
+
+        if (Maybe.isNothing(maybeProjectPathToOpen)) {
```

Comment:
I think it's better to use `foldMaybeWith` from `xod-fun-tools` instead of manually checking for nothingness and then exploding

Reply:
Good idea! 👍 
Done!